### DISCUSSION TITLE
What's the limit of the archived tasks?

### DISCUSSION THREAD

**Discussion by divyalok-zebpay** [hibiken/asynq#899] [General]
Hi folks!

We are using asynq on prod for one of our services and we've observed that archived tasks are capping at 9999. But, is this value configurable?

**Comment by amey09** [comment]
> Hi folks!
> 
> We are using asynq on prod for one of our services and we've observed that archived tasks are capping at 9999. But, is this value configurable?

@divyalok-zebpay you found anything on this? What can be done of archived tasks? From WebCLI as well they aren't getting deleted neither could run them. It shows 404 in network tab.
There isn't any documentation on how to deal with the archived tasks.

**Comment by kamikazechaser** [comment]
#978

**Reply by amey09** [reply]
thanks !
